### PR TITLE
Version 9.3.5 rc1

### PR DESCRIPTION
**Pull request by beenje**
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

Build conda package for cppTango 9.3.5-rc1 and add windows support.

This PR is to the `rc` branch and the package will be published to the `conda-forge/label/cpptango_rc` channel.

### CODE REVIEW

**Comment by beenje** [comment]
@conda-forge-admin, please rerender

**Comment by beenje** [comment]
@t-b, I'd appreciate if you could have a look at the `bld.bat` file to see if the options I use are correct.
I noticed this warning in the logs:

```
CMake Warning:
  Manually-specified variables were not used by the project:

    USE_PCH
```

Should I remove `USE_PCH`?

**Comment by t-b** [comment]
@beenje Sure.

recipe/bld.bat:

```
     -DCMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=TRUE ^
```

is not required anymore as that is now implicitly set when building the shared library version.

```
   -DUSE_PCH=OFF ^
```

This option does not exist in the 9.3-backports branch, but only in main. You can safely remove it.

For building I usually prefer cmake:

```
cmake --build . --config Release
cmake --build . --target install
```

but I guess nmake works as well.

**Comment by beenje** [comment]
Thanks @t-b! I removed those options.
Regarding `nmake`, I really don't have a strong preference. I took this from conda-forge doc as I don't have much windows knowledge: https://conda-forge.org/docs/maintainer/knowledge_base.html?#simple-cmake-based-bld-bat

**Comment by t-b** [comment]
@beenje If it's from the docs and works, I'd keep it.

**Comment by beenje** [comment]
> @beenje If it's from the docs and works, I'd keep it.

OK. Your `cmake` commands are probably equivalent, but it seems many conda recipes use `nmake` on windows...

**Comment by bourtemb** [comment]
Hi @beenje , you could actually update the API documentation link in the README.md file too for the >= 9.3.5 versions..
Replacing `https://tango-controls.github.io/cppTango-docs` with `https://tango-controls.gitlab.io/cppTango/`
We'll need to update the README.md file in cpptango repo as well with this new location.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -65,7 +80,7 @@ about:
   description: |
    This is the Tango-Controls C++ library (a.k.a. cppTango).
    Tango-Controls is a software toolkit for building control systems.
-  dev_url: https://github.com/tango-controls/cppTango
+  dev_url: https://gitlab.com/tango-controls/cppTango
   doc_url: https://tango-controls.github.io/cppTango-docs
   doc_source_url: https://github.com/tango-controls/cppTango-docs
```
Comment (bourtemb) [inline recipe/meta.yaml:85]:
```suggestion
  doc_url: https://tango-controls.gitlab.io/cppTango/
  doc_source_url: https://gitlab.com/tango-controls/cppTango
```
The source of the Doxygen documentation is actually in cppTango code now.
We have a CI/CD job triggered on tags to generate this doc.

File: README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ Feedstock license: [BSD-3-Clause](https://github.com/conda-forge/cpptango-feedst
 
 Summary: Tango-Controls C++ library
 
-Development: https://github.com/tango-controls/cppTango
+Development: https://gitlab.com/tango-controls/cppTango
 
 Documentation: https://tango-controls.github.io/cppTango-docs
```
Comment (bourtemb) [inline README.md:14]:
```suggestion
Documentation: https://tango-controls.gitlab.io/cppTango
```

**Comment by beenje** [comment]
@conda-forge-admin, please rerender

File: README.md (hunk 2 above)
Comment (beenje) [reply README.md:14]: To update the README, we should rerender. I should have after changing the doc url. I just did.

**Comment by bourtemb** [comment]
Now that we support the RelWithDebInfo CMAKE_BUILD_TYPE since 9.3.5-rc1 on Linux, it might be interesting to use it when building the conda package on Linux to improve the performances without losing the debug infos?

**Pull request merged**